### PR TITLE
Sync with CWL-Airflow 1.2.N

### PR DESCRIPTION
**Pull request by heylel-b-sh**
Should work with CWL-Airflow 1.2.N

Applied changes:
- refactored `post_build.sh` script. However, lines starting from **_138_** remained unchanged as their usage is not absolutely clear and cannot be properly refactored at the moment
- In `SatelliteApp.ts`
   - all CWL-Airflow related executables should be run from `bin_portable` folder
   - no need to copy `clean_dag_run.py` as it's created by `cwl-airflow init --upgrade` command, where `--upgrade` will additionally convert all DAGs to the new format
   - no need to call `airflow initdb/upgradedb` as `cwl-airflow init` does it
   - `process_report` connection doesn't require `--extra` parameter anymore
   - no need to set `PYTHONPATH` as all CWL-Airflow related executables are bash scripts that already set all required environment variables

### CODE REVIEW

File: electron/SatelliteApp.ts [hunk 1]
```diff
@@ -436,16 +436,16 @@ export class SatelliteApp {
...
             ROOT_URL: `${this.satelliteSettings.baseUrl}`,
             PORT: `${this.satelliteSettings.port}`,
             METEOR_SETTINGS: this.getSatelliteConf(),
             NODE_OPTIONS: '--trace-warnings --pending-deprecation',
             PATH: `${this.services_base_path}:${this.airflow_base_path}/bin_portable:/usr/bin:/bin:/usr/local/bin`
-        }
+        };
         if (this.proxySettings) {
-            env_var["https_proxy"] = `${this.proxySettings}`
+            env_var['https_proxy'] = `${this.proxySettings}`;
```
Comment (portah) [inline electron/SatelliteApp.ts]: As we discussed ```proxy``` should be part of ```this.satelliteSettings```. So the code should be ```this.satelliteSettings.proxy```

File: electron/SatelliteApp.ts [hunk 2]
```diff
@@ -436,16 +436,16 @@ export class SatelliteApp {
...
             MONGO_URL: `mongodb://localhost:${this.satelliteSettings.mongoPort}/scidap-satellite`,
             ROOT_URL: `${this.satelliteSettings.baseUrl}`,
             PORT: `${this.satelliteSettings.port}`,
             METEOR_SETTINGS: this.getSatelliteConf(),
             NODE_OPTIONS: '--trace-warnings --pending-deprecation',
             PATH: `${this.services_base_path}:${this.airflow_base_path}/bin_portable:/usr/bin:/bin:/usr/local/bin`
-        }
+        };
         if (this.proxySettings) {
-            env_var["https_proxy"] = `${this.proxySettings}`
```
Comment (portah) [inline electron/SatelliteApp.ts:455]: https & http

File: electron/SatelliteApp.ts [hunk 3]
```diff
@@ -615,9 +615,16 @@ export class SatelliteApp {
...
+        
+        // Need to make sure that clean_dag_run.py will be updated (init creates new if not present)
+        try {
+            await fs.promises.unlink(`${this.airflowSettings.AIRFLOW_HOME}/dags/clean_dag_run.py`)
+        } catch (e) {
+            Log.info('Failed to remove clean_dag_run.py');
+        }
 
         const init_commands = [
-            `cwl-airflow init`
```
Comment (portah) [inline electron/SatelliteApp.ts]: Do we need to replace init with init --update ? It's the first start no anything

File: electron/SatelliteApp.ts [hunk 4]
```diff
@@ -627,7 +634,7 @@ export class SatelliteApp {
         airflowConfig.core.load_examples = 'False';
         airflowConfig.core.hostname_callable = 'socket:gethostname';
         await fs.promises.writeFile(`${this.airflowSettings.AIRFLOW_HOME}/airflow.cfg`, stringify(airflowConfig, { whitespace: true }));
-        // await fs.promises.mkdir(`${this.airflowSettings.AIRFLOW_HOME}/dags`, { recursive: true });
+        await fs.promises.mkdir(`${this.airflowSettings.AIRFLOW_HOME}/dags`, { recursive: true });
         // await fs.promises.copyFile(`${this.airflow_base_path}/Resources/app/cwl_airflow/dags/clean_dag_run.py`, `${this.airflowSettings.AIRFLOW_HOME}/dags/clean_dag_run.py`);
```
Comment (portah) [inline electron/SatelliteApp.ts]: if we run just init with now --upgrade do we still need to copy clean dag?

File: electron/SatelliteApp.ts [hunk 5]
```diff
@@ -252,8 +252,7 @@ export class SatelliteApp {
                 PM2_API_PORT: this.satelliteSettings.pm2Port,
                 // PM2_HOME: this.pm2_home,
```
Comment (portah) [inline electron/SatelliteApp.ts:251]: maybe PM2 needs proxy too? it reports to home a cool staff of data

File: electron/SatelliteApp.ts (hunk 3 above)
Comment (heylel-b-sh) [reply electron/SatelliteApp.ts]: If there is no any DAGs in the dags folder `--upgrade` won't do anything, so you can leave just `cwl-airflow init`

File: electron/SatelliteApp.ts (hunk 4 above)
Comment (heylel-b-sh) [reply electron/SatelliteApp.ts]: It's not present in the latest commit

File: electron/SatelliteApp.ts (hunk 1 above)
Comment (heylel-b-sh) [reply electron/SatelliteApp.ts]: Done

File: electron/SatelliteApp.ts (hunk 2 above)
Comment (heylel-b-sh) [reply electron/SatelliteApp.ts:455]: Done

File: electron/SatelliteApp.ts (hunk 5 above)
Comment (heylel-b-sh) [reply electron/SatelliteApp.ts:251]: I would prefer to add it later when needed, but it's up to you

**Pull request merged**